suite-native/notifications/src/components/TransactionNotification.tsx

```diff
@@ -0,0 +1,143 @@
+import React from 'react';
+import { useSelector } from 'react-redux';
+
+import { useNavigation } from '@react-navigation/native';
+
+import { Box, Text } from '@suite-native/atoms';
+import { AccountKey, TransactionType } from '@suite-common/wallet-types';
+import { TransactionIcon } from '@suite-native/transactions';
+import { CryptoAmountFormatter, AccountAddressFormatter } from '@suite-native/formatters';
+import {
+    TransactionsRootState,
+    selectTransactionByTxidAndAccountKey,
+    selectTransactionFirstTargetAddress,
+} from '@suite-common/wallet-core';
+import {
+    RootStackRoutes,
+    StackNavigationProps,
+    RootStackParamList,
+} from '@suite-native/navigation';
+import { Icon } from '@trezor/icons';
+import { NetworkSymbol } from '@suite-common/wallet-config';
+import { useNativeStyles, prepareNativeStyle } from '@trezor/styles';
+
+import { AnnouncementNotification } from './AnnouncementNotification';
+
+type TransactionNotificationProps = {
+    txid: string;
+    accountKey: AccountKey;
+    isHiddenAutomatically?: boolean;
+};
+
+type TransactionNotificationDescriptionProps = {
+    amount: string | number;
+    transactionType: TransactionType;
+    networkSymbol: NetworkSymbol;
+    targetAddress?: string;
+};
+
+type TransactionTypeProperties = {
+    title?: string;
+    preposition?: string;
+};
+
+const addressContainerStyle = prepareNativeStyle(_ => ({
+    maxWidth: '35%',
+}));
+
+const transactionTypeToContentMap = {
+    recv: { title: 'Incoming transaction', preposition: 'from' },
```

Comment:
nit: `preposition` sounds really strange :D

Reply:
renamed to `prefix` in [40bdfc5](https://github.com/trezor/trezor-suite/pull/7685/commits/40bdfc547bec9d4972f293bc2fa699868a835f00)